### PR TITLE
add single pages for adopters

### PR DESCRIPTION
**Pull request by SAMZONG**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Add 'adopters' as a top-level page, without the need to follow document version updates.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by SAMZONG** [comment]
localhost_3000_adopters_(墨刀)

**Comment by SAMZONG** [comment]
@Poor12 @RainbowMango 

visit https://deploy-preview-414--karmada.netlify.app/

add Adopters to navgation and footer.

**Review by RainbowMango — COMMENTED** [review]
Glad to see that you made it!
Thanks.

**Comment by Poor12** [comment]
Hi @SAMZONG, what we want to put on top of website is [this page](https://karmada.io/docs/casestudies/adopters).

**Comment by SAMZONG** [comment]
> Hi @SAMZONG, what we want to put on top of website is [this page](https://karmada.io/docs/casestudies/adopters).

Okay, I will change this.

**Comment by SAMZONG** [comment]
@Poor12 it's already updated to correct content.

File: i18n/zh/docusaurus-plugin-content-pages/adopters.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Karmada Adopters
+hide_table_of_contents: true
+---
+
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-pages/adopters.md:5]: I don't know why I can't see the title.

Add a title by `# Karmada Adopters`?

File: i18n/zh/docusaurus-theme-classic/footer.json [hunk 2]
```diff
@@ -23,6 +23,10 @@
     "message": "博客",
     "description": "The label of footer link with label=Blog linking to blog"
   },
+  "link.item.label.Adopters": {
+    "message": "案例",
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-theme-classic/footer.json:27]:
```suggestion
    "message": "采用者",
```
？ 将来可能还需要加入 `Case Study` ,届时可能会用到 `案例` 一词。

File: src/pages/adopters.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Karmada Adopters
+hide_table_of_contents: true
+---
+
```
Comment (RainbowMango) [inline src/pages/adopters.md:5]: ditto.

**Review by RainbowMango — COMMENTED** [review]
Generally looks good, thanks for doing this. Just some nits.

File: i18n/zh/docusaurus-theme-classic/footer.json (hunk 2 above)
Comment (SAMZONG) [reply i18n/zh/docusaurus-theme-classic/footer.json:27]: agree with u suggestion. and I was updated.

Comment (RainbowMango) [reply i18n/zh/docusaurus-theme-classic/footer.json:27]: 我看到预览版本已经改过了，这里`案例`两字不用改吗？

Comment (SAMZONG) [reply i18n/zh/docusaurus-theme-classic/footer.json]: footer 加在页面底部，抱歉，遗漏了一并修改啦.

Comment (RainbowMango) [reply i18n/zh/docusaurus-theme-classic/footer.json]: No worries.

**Pull request merged**